user-guide/scheduler.rst

```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
 Resource Limits
 ---------------
 
-There are different resource limits on ARCHER2 for different purposes.
+The ARCHER2 resource limits for any given job are covered by three separate attributes.
+
+* The amount of *primary resource* you require (i.e., CPU cores).
+* The *partition* that you want to use - this specifies the nodes that are eligible to run your job.
+* The *Quality of Service (QoS)* that you want to use - this specifies the job limits that apply.
+
+Each of these attributes is described in more detail below. 
+
+The *primary resources* you request are *compute* resources: CPU cores on the compute nodes. Other node resources,
+such as memory, are assigned pro rata based on the primary resource that you request.
+
+.. warning::
+
+   On ARCHER2, you cannot specify the memory for a job using the ``--mem`` options to Slurm
+   (e.g. ``--mem``, ``--mem-per-cpu``, ``--mem-per-gpu``). The amount of memory you are 
+   assigned is calculated from the amount of primary resource you request.
+
+Primary resources on (CPU) compute nodes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The *primary resource* you request on compute nodes is the CPU core. The maximum amount of memory you are allocated is computed
+as the number of CPU cores you requested multiplied by 1/128th of the total memory available (as there are 128 CPU cores per node).
+So, if you request the full node (128 cores), then you will be allocated all of the memory (256 GB) available on a standard compute node;
+however, if you request 1 core, then you will be assigned a maximum of 256/128 = 2 GB of the memory available on the node
+(for high memory nodes the memory per core is 512/128 = 4 GB).
 
 .. note::
 
-   Details on the resource limits will be added when the ARCHER2 system
-   is available.
+   Using the ``--exclusive`` option in jobs will give you access to the full node memory even
+   if you do not explicitly request all of the CPU cores on the node.
+
```

Comment:
See above, this does not apply on ARCHER2

Reply:
OK